where-to-travel/src/main/webapp/script.js

```diff
@@ -259,13 +259,12 @@ function addPlacesFromDirection(lat, lng, place_candidates) {
 
         function callback(response, status) {
           if (status == 'OK') {
-            const origins = response.originAddresses;
             for (row of responce.rows) {
-              const results = response.rows[row].elements;
+              const results = row.elements;
               for (result of results) {
```

Comment:
Should this be "for(i = 0; i < results.length; i ++)" for the indexing on lines 269 and 270?

Reply:
No because I want to use the index to get the information form listPlaces